src/Faker/Provider/fa_IR/Internet.php

```diff
@@ -28,18 +28,18 @@ class Internet extends \Faker\Provider\Internet
     protected static $lastNameAscii = array(
         'abdullahi', 'abbasi', 'abedini', 'ahadi', 'ahmadi', 'akbari', 'alizadeh', 'asadi',
         'baraghani', 'babaei',
-        'chamran',
+        'chamran', 'chabahar',
```

Comment:
Please don't add names at random places. We conform to the datasource (cited in the phpDoc above)